### PR TITLE
TFSID952474: Disable Aggregate:MoveGroupBysToMeasures since FuncallRewrite:RewriteConstantFuncall is also disabled

### PR DESCRIPTION
**Pull request by LingyanYin**
We have completed a new feature planned to turn on for 19.4. There is a rewrite rule added which has to work with RewriteConstantFuncall which is disabled in TDVT tests, so we also need to disable this one since it would cause some SQL text unable to be executed in DB2.

### CODE REVIEW

**Comment by LingyanYin** [comment]
> Thanks for the PR Lingyan, let's wait for a review from Logan but if he gives the thumbs up on the change it'd be helpful if you could fix the unit tests it looks like this change would break. You can see them in Travis CI: https://travis-ci.org/tableau/connector-plugin-sdk/jobs/566088933

Yeah. I took a look and it seems some changes in command line don't match the expected since I added another disabled rewrite in the command. I will check to see how to update those expected ones.

**Comment by lukewrites** [comment]
> Yeah. I took a look and it seems some changes in command line don't match the expected since I added another disabled rewrite in the command. I will check to see how to update those expected ones.

I _think_ it'll be a quick fix since those tests are just string comparisons, but if they get weird ping me and I'm happy to help.

**Comment by LingyanYin** [comment]
close it and discard it since Logan suggested another way of doing this

File: tdvt/tdvt/tabquery.py [hunk 1]
```diff
@@ -3,6 +3,7 @@
 
 from .resources import *
 tab_cli_exe = ''
+logical_rewrite_appoption = ''
```
Comment (lriggs) [inline tdvt/tdvt/tabquery.py]: I think it would be simpler if you took this out, then you wouldn't need the else clause on line 81 either.

File: tdvt/test/tdvt_test.py [hunk 2]
```diff
@@ -253,9 +253,9 @@ def test_command_line_full(self):
         cmd_line = build_tabquery_command_line_local(work)
         cmd_line_str = ' '.join(cmd_line)
         if sys.platform in ('win32', 'cygwin'):
-            expected = 'tabquerytool.exe --expression-file-list my/output/dir\mytest\\tests.txt -d mytds.tds --combined --output-dir my/output/dir -DLogDir=my/output/dir\mytest -DOverride=ProtocolServerNewLog -DLogLevel=Debug -DLogicalQueryRewriteDisable=Funcall:RewriteConstantFuncall'  # noqa: E501
+            expected = 'tabquerytool.exe --expression-file-list my/output/dir\mytest\\tests.txt -d mytds.tds --combined --output-dir my/output/dir -DLogDir=my/output/dir\mytest -DOverride=ProtocolServerNewLog -DLogLevel=Debug ' + logical_rewrite_appoption  # noqa: E501
```
Comment (lriggs) [inline tdvt/test/tdvt_test.py]: Can't you just leave the test file like it is (ie revert these changes)?
